### PR TITLE
Document how to avoid non-free dependencies on Android

### PR DESCRIPTION
**Pull request by TheLastProject**
Given #288 was closed with the decision to not provide an explicit libre flavor because excluding the non-free dependencies is very simple, I am hoping documenting it like this should help users be aware that this is an option.

Thanks for considering :)

### CODE REVIEW

**Review by wipfli — COMMENTED** [review]
Looks like a good idea to me, thanks. @maxammann can you maybe comment if this is correct?

File: README.md [hunk 1]
```diff
@@ -58,6 +58,13 @@ We thank everyone who supported us financially in the past and special thanks to
 
 3. Sync gradle and rebuild your app
 
+*Note: MapLibre by default ships with the proprietary Google Play Location Services. If you want to avoid pulling proprietary dependencies into your project, you can exclude Google Play Location Services as follows:*
+```gradle
+    implementation ('org.maplibre.gl:android-sdk:9.4.0') {
```
Comment (wipfli) [inline README.md:63]:
```suggestion
    implementation ('org.maplibre.gl:android-sdk:<version>') {
```

**Review by wipfli — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (wipfli) [reply README.md:63]: Like that we don't have to update the readme at every version...

Comment (TheLastProject) [reply README.md:63]: __

Comment (TheLastProject) [reply README.md:63]: I guess you want that changed on line 54 then too?

Comment (wipfli) [reply README.md:63]: Yes good catch. Can you update it?

**Pull request merged**